service/src/main/java/uk/gov/hmcts/reform/fpl/service/document/ManageDocumentService.java

```diff
@@ -134,10 +136,15 @@
                 Optional<Element<HearingFurtherEvidenceBundle>> bundle = findElement(selectedHearingId, bundles);
 
                 if (bundle.isPresent()) {
-                    return getUserSpecificSupportingEvidences(bundle.get().getValue().getSupportingEvidenceBundle());
+                    List<Element<SupportingEvidenceBundle>> evidenceBundle
```

Comment:
maybe easier and better would be to add to SupportingEvidenceBundle method 

    //For backward compatibility
    public FurtherEvidenceType getType() {
        return Optional.ofNullable(type).orElse(OTHER_REPORTS);
    }
    
    and it should automagically do the thing for old bundles

Reply:
much better

Reply:
actually this isnt quite right, it will apply it for all but the bundle is used for C2 supporting + correspondence too, in this case we don't want a further evidence type to be applied